herd/tests/instructions/AArch64.mixed/M000.litmus

```diff
@@ -0,0 +1,14 @@
+AArch64 M000
+(* Does not work yet without option -debug mixed... *)
```

Comment:
Should this work in non-mixed mode eventually, or is it fundamentally a mixed-mode test?

Reply:
This is fundamentally a mixed-size test and it would be a good thing to have tests specific to mixed-size. Unfortunately, this test is not ready yet. There is another, unrelated, shortcoming.